.github/workflows/install_ramalama.yml

```diff
@@ -0,0 +1,41 @@
+name: Install RamaLama
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  install_ramalama:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]  # Runs on Ubuntu, macOS, and Fedora
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set Up Dependencies (Ubuntu)
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -y lshw curl
+
+      - name: Set Up Dependencies (macOS)
+        if: matrix.os == 'macos-latest'
+        run: |
+          if ! command -v brew &> /dev/null; then
+            echo "Homebrew is required but not found. Please install it."
+            exit 1
+          fi
+          brew install lshw curl || true
+
+      - name: Run RamaLama Installer
+        run: |
+          chmod +x install.sh
+          ./install.sh
```

Comment:
We probably want to do:

```
./install.sh -l
```

here.


```
./install.sh
```

alone doesn't make a while pile of sense because it's pull from "s" branch, we want to test the current branch rather than the "s" branch.